### DISCUSSION TITLE
how to use allowNonExtraProperties on lucid?

### DISCUSSION THREAD

**Discussion by rikoefendi** [adonisjs/core#1478] [Help]
I found a code
public fill(values: any, allowNonExtraProperties: boolean = false): this { this.$attributes = {} this.merge(values, allowNonExtraProperties) this.fillInvoked = true return this }

and my question, how to use allowNonExtraProperties on event create data?

*i'm so sorry if my question not easy understanding

**Comment by thetutlage** [comment]
Follow this [markdown guide](https://guides.github.com/features/mastering-markdown/) to format the code examples please.
